packages/styles/scss/components/_richtext.scss

```diff
@@ -1,68 +1,72 @@
 @use "../tokens" as *;
 @use "../functions" as *;
+@import "./blockquote";
 @import "../mixins";
 
 .ilo--richtext {
```

Comment:
The same goes there, as mentioned in #999 if we want to expose anything to end user we should define them as a component-scoped variables, and let's remove `scss`  ones,  like `$color-font-caption`

Reply:
@GGKapanadze not sure what you're referring to here.

Reply:
Oh wait no I see it, good catch! @GGKapanadze

Reply:
- There are still some variables defined as `SCSS` ones 
 - If we want expose any `richtext` scoped variables this is a good place 

but I think we can do it in scope of #1240